context/package.json

```diff
@@ -96,7 +101,7 @@
     "babel-plugin-styled-components": "^1.11.1",
     "chromatic": "^5.9.2",
     "clean-webpack-plugin": "^3.0.0",
```

Comment:
Can we uninstall `clean-webpack-plugin` after the changes to our webpack config files?